### PR TITLE
Update docs about querying saved searches

### PR DESCRIPTION
**Pull request by DavidPerezIngeniero**
https://github.com/quodlibet/quodlibet/issues/2238

### CODE REVIEW

File: quodlibet/docs/guide/searching.rst [hunk 1]
```diff
@@ -203,3 +203,19 @@ name ``"Sigur Rós"``.
 
 
 Now you can search anything!
+
+
+Reusing queries
+---------------
+
+Complex queries can be splitted into simpler ones.  Also, a query can be reused
```
Comment (lazka) [inline quodlibet/docs/guide/searching.rst]: splitted->split

File: quodlibet/docs/guide/searching.rst [hunk 2]
```diff
@@ -203,3 +203,19 @@ name ``"Sigur Rós"``.
 
 
 Now you can search anything!
+
+
+Reusing queries
+---------------
+
+Complex queries can be splitted into simpler ones.  Also, a query can be reused 
+in other ones.  In this way it is easier to change and administer your searches.
```
Comment (lazka) [inline quodlibet/docs/guide/searching.rst]: In this way -> This way

File: quodlibet/docs/guide/searching.rst [hunk 3]
```diff
@@ -203,3 +203,19 @@ name ``"Sigur Rós"``.
...
+
+In order to do so, the ``Include Saved Search`` plugin must be activated.  
+If you create a saved search named ``Unrated`` you can search for unrated songs from the Beatles
+like this:
+
+``@(saved: Unrated) AND Beatles``
```
Comment (lazka) [inline quodlibet/docs/guide/searching.rst]: Do you mean ``&(@(saved: Unrated), Beatles)`` ?

Comment (DavidPerezIngeniero) [inline quodlibet/docs/guide/searching.rst]: That's true. I'm using other search engines, and I have confused the syntax.  Sorry

**Comment by lazka** [comment]
Thanks!

**Pull request merged**